packages/fast-components-react-base/src/hypertext/hypertext.props.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+/**
+ * Hypertext HTML tags
+ */
+export enum HypertextHTMLTags {
+    a = "a",
+}
+
+export interface IHypertextProps {
+
+    id?: string;
+
+    text?: string;
+
+    href?: string;
+
+    ariaLabel?: string;
```

Comment:
based on our conversation yesterday with Mat, I think this is the exception here and likely wouldn't expect this as a first-class prop